modules/efk-logging-fluentd-json.adoc

```diff
@@ -41,8 +45,8 @@ If you set the `MERGE_JSON_LOG` and `CDM_UNDEFINED_TO_STRING` enviroment variabl
 
```

Comment:
@richm Does this problem affect Rsyslog as it affected Fluentd? 
`If you set the `MERGE_JSON_LOG` and `CDM_UNDEFINED_TO_STRING` enviroment variables to `true`, you might receive an Elasticsearch *400* error. `

Reply:
Yes.